core/compilers.ts

```diff
@@ -23,7 +23,7 @@ export function compile(code: string, path: string): string {
   if (path.endsWith(".yaml")) {
     try {
       const yamlAsJson = JSON.stringify(loadYaml(code));
-      return `exports.asJson = () => (${yamlAsJson})`;
+      return `exports.asJson = ${yamlAsJson}`;
```

Comment:
why not just `exports = ${yamlAsJson}`?

Reply:
When setting `exports = ...` directly, the return value of `require()` always ends up being `{}`.

The wrapper looks like:

```
(function (exports, require, module, __filename, __dirname) { "use strict"; <return code here> })
```

Reply:
It occurred to me: the right way to do this would be:

module.exports = fileAsJson